rules/typescript.js

```diff
@@ -45,6 +45,13 @@ const typescriptRules = {
   '@typescript-eslint/no-parameter-properties': 0,
   '@typescript-eslint/no-triple-slash-reference': 2,
   '@typescript-eslint/no-unnecessary-type-assertion': 2,
+  '@typescript-eslint/no-unused-vars': [
+    2,
+    {
+      ignorRestSiblings: true,
```

Comment:
misspelled

Reply:
I don't now what you're talking about, I'm great at spelling.